hoomd/md/EvaluatorPairZBL.h

```diff
@@ -40,15 +41,76 @@ class EvaluatorPairZBL
 {
     public:
         //! Define the parameter type used by this pair potential evaluator
-        typedef Scalar2 param_type;
+        struct param_type
```

Comment:
It appears that once we update Moliere to the new format this class can just reuse that type. The alternate aF computation will need to be listed in the documentation or provided as a convenience method.